src/linker/Linker.Steps/RemoveResourcesStep.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Linq;
+using Mono.Cecil;
+
+namespace Mono.Linker.Steps
+{
+	public class RemoveResourcesStep : BaseStep 
+	{
+		protected override void ProcessAssembly (AssemblyDefinition assembly)
+		{
+			if (!assembly.MainModule.HasResources) return;
+			RemoveFSharpCompilationResources (assembly);
```

Comment:
Please add a check which enables this only for assemblies which are linked or saved